src/screens/CreatePost/elements/Card.js

```diff
@@ -1,51 +1,52 @@
 import React from 'react';
-import {View, StyleSheet, Platform} from 'react-native';
-import {COLORS} from '../../../utils/theme';
-
-const Card = (props) => {
+import { View, StyleSheet, Platform } from 'react-native';
+import { colors } from '../../../utils/colors';
+const Card = props => {
   return (
     // <View style={{ ...styles.card, ...props.style }}></View>
     <View // Parent
       style={{
         // No backgroundColor
-        shadowColor: COLORS.black,
-        shadowOffset: {width: 0, height: 1},
+        shadowColor: '#000',
+        shadowOffset: { width: 0, height: 1 },
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `shadowColor` property has been changed from `COLORS.black` to `'#000'`. This change should be fine as long as the color codes represent the same color.

```diff
- shadowColor: COLORS.black,
+ shadowColor: '#000',
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->